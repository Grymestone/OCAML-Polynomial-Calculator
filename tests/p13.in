6a+2a^33
