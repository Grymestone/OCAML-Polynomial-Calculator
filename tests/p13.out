((6*a)+(2*(a)^33))
6x^1 + 2x^33